spring-framework/3.2.9.wso2v2/pom.xml

```diff
@@ -0,0 +1,98 @@
+    <groupId>org.springframework.ws.wso2</groupId>
+    <artifactId>spring.framework</artifactId>
+    <packaging>bundle</packaging>
+    <name>spring.framework.wso2</name>
+    <version>3.2.9.wso2v2</version>
+    <description>
+        This bundle will export packages from spring framework
+    </description>
+    <url>http://wso2.org</url>
+
+   <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>WSO2 Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+   </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.ws</groupId>
+            <artifactId>spring-xml</artifactId>
+            <version>2.1.4.RELEASE</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>${version.spring}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>${version.spring}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${version.spring}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>${version.spring}</version>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>1.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.springframework.*; version=${exp.pkg.version.spring},
+                        </Export-Package>
+                        <Import-Package>
+                            !org.springframework.*; version=${exp.pkg.version.spring},
+                            javax.xml.stream.*; version="1.0.1",
+                            javax.xml.soap; version="0.0.0",
+                            *;resolution:=optional
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Do we really want dynamic import.
i.e. do we use "class.forName()" within "spring.framework"?

Reply:
I think we don't use class.forName() within "spring.framework"? Since this is an orbit and dynamic import was there for a long we will skip this at the moment. We will address this with https://github.com/wso2/carbon-kernel/issues/2582